src/hotspot/share/gc/g1/g1Policy.cpp

```diff
@@ -669,8 +667,13 @@ bool G1Policy::should_retain_evac_failed_region(uint index) const {
   return live_bytes < threshold;
 }
 
-void G1Policy::record_young_collection_start() {
+void G1Policy::record_gc_pause_start_time() {
```

Comment:
Could this be named `record_pause_start_time()` which would better correspond to the `record_pause` call at the end of the respective pause?

There is also a kind of mismatch to this name and the `record_cur_collection_start_sec`, which might be more appropriately named `record_cur_pause_start_sec` following this naming.

It's just right now the code uses `gc`, `collection`, `pause` in the names which makes reading a bit harder than necessary.